src/dependencies.jl

```diff
@@ -884,7 +884,11 @@ macro install(_libmaps...)
                         println(depsfile_buffer)
                         println(depsfile_buffer, "# Load-hooks")
                         println(depsfile_buffer, join(load_hooks,"\n"))
-                        depsfile_content = chomp(String(take!(depsfile_buffer)))
+                        if VERSION < v"0.5-" # remove this case after dropping 0.4 support
```

Comment:
`take!` on a buffer is much more recent than this, and need to be specific with VERSION cutoffs